buildconfig/CMake/LinuxPackageScripts.cmake

```diff
@@ -214,11 +239,20 @@ execute_process(COMMAND "chmod" "+x" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/AddPytho
 # Package version
 unset(PYTHON_ARGS)
 
-# used by mantidplot and mantidworkbench
-set(LOCAL_PYPATH "\${INSTALLDIR}/bin:\${INSTALLDIR}/lib:\${INSTALLDIR}/plugins")
+# used by mantidworkbench
+if(CONDA_BUILD)
+  set(LOCAL_PYPATH "\${CONDA_PREFIX}/bin:\${CONDA_PREFIX}/lib:\${CONDA_PREFIX}/plugins")
+  set(PYTHON_EXEC_LOCAL "\${CONDA_PREFIX}/bin/python")
+  set(PREAMBLE "${CONDA_PREAMBLE_TEXT}")
+  set(MANTIDWORKBENCH_EXEC "\${CONDA_PREFIX}/bin/workbench") # what the actual thing is called
+else()
+  set(LOCAL_PYPATH "\${INSTALLDIR}/bin:\${INSTALLDIR}/lib:\${INSTALLDIR}/plugins")
+  set(PYTHON_EXEC_LOCAL "${PYTHON_EXECUTABLE}")
```

Comment:
Should this be `set(PYTHON_EXEC_LOCAL "\${PYTHON_EXECUTABLE}")` so that the `PYTHON_EXECUTABLE` is expanded later?

Reply:
`PYTHON_EXECUTABLE` is the value from cmake so it should be expanded here.